<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcul_module</class>
 <widget class="QWidget" name="calcul_module">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALCUL_DU_MODULE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../PFE/INTERFACE/biblio/ui/reducteur_gearexpert.gif</normaloff>../PFE/INTERFACE/biblio/ui/reducteur_gearexpert.gif</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>466</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Valeurs d'entrée</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_48">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Rpe</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label2">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Résistance pratique à l'extension </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_47">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>K</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label1">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Coefficient de largeur de denture </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_31">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>T</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" alignment="Qt::AlignHCenter">
            <widget class="QLineEdit" name="lineEdit1">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Effort tangentiel sur la dent </string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" alignment="Qt::AlignHCenter">
            <widget class="QLineEdit" name="lineEdit3">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="0" column="2" alignment="Qt::AlignHCenter">
            <widget class="QLineEdit" name="lineEdit">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label3">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Angle d'hélice</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>β</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" alignment="Qt::AlignHCenter">
            <widget class="QLineEdit" name="lineEdit2">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_1">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Aide</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>CALCULEZ</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>SAUVEGARDER LES RESULTATS DU CALCUL</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_45">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Valeurs de sortie</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label4">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>La valeur normalisée du module </string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLineEdit" name="lineEdit4">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_2">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>RETOUR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
